### PR TITLE
NEXT-14332/3 Add documentation to docs

### PR DESCRIPTION
**Pull request by J-Rahe**
These [Pr's](https://github.com/shopware/development/pull/141) have added support for bundled private composer dependencies, this pr adds an short docs entry for that.

### CODE REVIEW

File: guides/plugins/plugins/plugin-fundamentals/using-composer-dependencies.md [hunk 1]
```diff
@@ -83,6 +83,28 @@ class MySubscriber implements EventSubscriberInterface
 ```
 {% endcode %}
 
+## Adding private composer dependencies
+
+You can bundle composer dependencies with your Plugin by adding them to the /packages/ folder of your plugin.
```
Comment (PaddyS) [inline guides/plugins/plugins/plugin-fundamentals/using-composer-dependencies.md:88]:
```suggestion
You can bundle composer dependencies with your plugin by adding them to the `/packages/` folder of your plugin.
```

File: guides/plugins/plugins/plugin-fundamentals/using-composer-dependencies.md [hunk 2]
```diff
@@ -83,6 +83,28 @@ class MySubscriber implements EventSubscriberInterface
...
+
+example structure:
```
Comment (PaddyS) [inline guides/plugins/plugins/plugin-fundamentals/using-composer-dependencies.md:90]:
```suggestion
Example structure:
```

File: guides/plugins/plugins/plugin-fundamentals/using-composer-dependencies.md [hunk 3]
```diff
@@ -83,6 +83,28 @@ class MySubscriber implements EventSubscriberInterface
...
+````text
+MyAwesomePlugin
```
Comment (PaddyS) [inline guides/plugins/plugins/plugin-fundamentals/using-composer-dependencies.md:92]:
```suggestion
SwagBasicExample
```

File: guides/plugins/plugins/plugin-fundamentals/using-composer-dependencies.md [hunk 4]
```diff
@@ -83,6 +83,28 @@ class MySubscriber implements EventSubscriberInterface
...
+├── packages
+│   └── my-private-dependency/
+│       ├── composer.json
+│       └── src/
+│           └── SomeCoolService.php
+├── src/
+│   └── MyAwesomePlugin.php
```
Comment (PaddyS) [inline guides/plugins/plugins/plugin-fundamentals/using-composer-dependencies.md:99]:
```suggestion
│   └── SwagBasicExample.php
```

**Review by PaddyS — CHANGES_REQUESTED** [review]

**Comment by PaddyS** [comment]
Also please fix commit message, kinda broken
"NEXT-12345 - Message"

**Review by shyim — APPROVED** [review]

File: guides/plugins/plugins/plugin-fundamentals/using-composer-dependencies.md (hunk 1 above)
Comment (PaddyS) [reply guides/plugins/plugins/plugin-fundamentals/using-composer-dependencies.md]: Still missing the packages part

Comment (J-Rahe) [reply guides/plugins/plugins/plugin-fundamentals/using-composer-dependencies.md]: missed that.

**Review by JoshuaBehrens — APPROVED** [review]

**Pull request merged**